otwcode/otwarchive
# AO3-6162 Remove absolute links in help pop-ups

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6162

## Purpose

What does this PR do?

What does this PR do?

Removes all references to outdated /archive_faqs hyperlinks in help pop-ups related to posting new work (this string doesn't appear in any other locations). Also uses relative linking rather than absolute.

The files changed are:

- additional-tags-help.html
- categories-help.html
- characters-help.html
- relationships-help.html
- fandom-help.html

## Testing Instructions

How can the Archive's QA team verify that this is working as you intended?

- Log in and go to the Post New Work page
- Select the blue ? icon best to the Fandom, Categories, Relationships, Characters, and Additional Tags labels
- Hover over the "Tags on the Archive FAQ" and confirm it is pointing to test.archiveofourown.org/faq/tags
- Select the "Tags on the Archive FAQ" hyperlink (or Tags FAQ link in the Fandom help)
- Confirm you are now on the Tags FAQ page
- Repeat for the remaining three labels

## References

Not to my knowlege

## Credit

*What name do you want us to use to credit your work in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?*

Sammie Louise

*What pronouns do you prefer (she/her, he/him, zie/hir etc)?*

she/her
